ProjectSettings/EditorBuildSettings.asset

```diff
@@ -4,11 +4,5 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes:
-  - enabled: 1
-    path: Assets/Scenes/StartScreen.unity
-    guid: 85d359e1133588742ab242de01e601ff
-  - enabled: 1
-    path: Assets/Scenes/LivingRoom.unity
-    guid: 1974aa80b246e0a41ba1fa3b2696791a
+  m_Scenes: []
```

Comment:
Pretty sure this will break the main menu screen. Remember you can manually chose which files to commit, always good to double check you want to keep the changes